docs/part-4-labeling-the-data-and-retrain/chapter-18-setup-label-studio.md

```diff
@@ -1,6 +1,204 @@
 # Chapter 18 - Setup Label Studio
 
-!!! warning "This is a work in progress"
+## Introduction
 
-    This chapter is a work in progress. Please check back later for updates. Thank
-    you!
+In the previous chapters, we have successfully deployed and accessed our model
+on Kubernetes, set up continuous deployment with a CI/CD pipeline, and trained
+the model on a Kubernetes pod. Now, we will focus on labeling new data to
+further improve our model's performance.
+
+[Label Studio](https://labelstud.io) is an open-source data labeling tool that
+supports various data types, including text, images, audio, and video. In this
+chapter, we will guide you through the process of setting up Label Studio in
+your environment. This includes installing the necessary dependencies,
+configuring the tool, and preparing it for data labeling tasks.
+
+By the end of this chapter, you will have a fully functional Label Studio
+instance ready to label new data, which will be used in subsequent chapters to
+retrain and enhance your model.
+
+## Steps
+
+### Download the Data
+
+Before configuring Label Studio, you will need to download additional data used
+for labeling.
+
+```sh title="Execute the following command(s) in a terminal"
+# Download the archive containing the extra data
+wget https://github.com/swiss-ai-center/a-guide-to-mlops/archive/refs/heads/extra-data.zip -O extra-data.zip
+```
+
+The downloaded archive must be decompressed and renamed:
+
+```sh title="Execute the following command(s) in a terminal"
+# Extract the dataset
+unzip extra-data.zip
+
+# Rename to the folder to `extra-data`
+mv a-guide-to-mlops-extra-data/ extra-data/
+
+# Remove the archive and the directory
+rm extra-data.zip
+```
+
+Finally, add the `extra-data` folder to the `.gitignore` file:
+
+```sh title="Execute the following command(s) in a terminal"
+# Add the `extra-data` folder to the `.gitignore` file
+echo "\n# Label Studio\nextra-data/" >> .gitignore
```

Comment:
Using `"` here means string litteral will be included. The new line character are included as literral "\n" indeed.

Either pass "-e" to the echo command, or adjust the `.gitignore` file manually as we did in previous chapters.

Note: this might be a difference of shell implementation too.